lib/synced/strategies/full.rb

```diff
@@ -43,48 +43,36 @@ class Full
       # @option options [Array|Hash] globalized_attributes: A list of attributes
       #   which will be mapped with their translations.
       def initialize(model_class, options = {})
-        @model_class           = model_class
-        @scope                 = options[:scope]
-        @id_key                = options[:id_key]
-        @data_key              = options[:data_key]
-        @only_updated          = options[:only_updated]
-        @include               = options[:include]
-        @local_attributes      = synced_attributes_as_hash(options[:local_attributes])
-        @api                   = options[:api]
-        @mapper                = options[:mapper].respond_to?(:call) ?
-                                   options[:mapper].call : options[:mapper]
-        @fields                = options[:fields]
-        @remove                = options[:remove]
-        @associations          = Array.wrap(options[:associations])
-        @association_sync      = options[:association_sync]
-        @perform_request       = options[:remote].nil? && !@association_sync
-        @remote_objects        = Array.wrap(options[:remote]) unless @perform_request
-        @globalized_attributes = synced_attributes_as_hash(options[:globalized_attributes])
-        @query_params         = options[:query_params]
+        @model_class                      = model_class
```

Comment:
don't shift stuff like that in PRs, it obscures the changes, if you want to restyle some code do it in a separate pr first